### PR TITLE
SwinMM/Initialize the SwinMM project

### PR DESCRIPTION
**Pull request by Zi-hao-Wei**
Hi, I add the SwinMM project into this repository.

### CODE REVIEW

File: SwinMM/WORD/dataset/dataset12_WORD_0.1.json
Comment (wyli) [inline SwinMM/WORD/dataset/dataset12_WORD_0.1.json:1]: thanks for the PR! could you please zip the json files and put them in a google drive folder? then you can provide instructions for the users to download them like this example https://github.com/Project-MONAI/research-contributions/tree/main/SwinUNETR/BRATS21#models

File: SwinMM/WORD/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Note for SwinMM Finetuning
+
+## Training
+
+### FIXME(outdated)
```
Comment (wyli) [inline SwinMM/WORD/README.md:5]: I guess this would need some updates

**Review by wyli — APPROVED** [review]
thanks for the PR, it looks great, please help fix those two minor issues

File: SwinMM/WORD/README.md (hunk 1 above)
Comment (Zi-hao-Wei) [reply SwinMM/WORD/README.md:5]: This file is out of date and I have removed it. All the training details can be find in the TRAINING.md

**Comment by Zi-hao-Wei** [comment]
Hi, I have fix the problems. Could you please check it again? Thanks.

**Comment by tangy5** [comment]
This is great, thank you.

**Comment by tangy5** [comment]
This PR looks good to me:
HI @Zi-hao-Wei can you double check:
1. Whether there are duplicated or unused lines of code in your scripts.
2. Whethere there are additional license. 
3. I saw the WORD_finetuned.pt is actually the BTCV finetined checkpoint download link, is that correct? 
4. Can you add MONAI paper reference to the README.md? 
```
@article{cardoso2022monai,
  title={Monai: An open-source framework for deep learning in healthcare},
  author={Cardoso, M Jorge and Li, Wenqi and Brown, Richard and Ma, Nic and Kerfoot, Eric and Wang, Yiheng and Murrey, Benjamin and Myronenko, Andriy and Zhao, Can and Yang, Dong and others},
  journal={arXiv preprint arXiv:2211.02701},
  year={2022}
}
```

Thanks

**Review by tangy5 — APPROVED** [review]

**Comment by Zi-hao-Wei** [comment]
Thanks for your reply. To address your concerns:
1. No duplicated or unused lines in the scripts.
2. No additional license.
3. This is a naming error and I have fixed the sharing link. Thank you for pointing out.
4. I have added it to the Citation part in README.md

File: SwinMM/Pretrain/utils/dataset_in_memory.py [hunk 2]
```diff
@@ -0,0 +1 @@
+../../WORD/utils/dataset_in_memory.py
```
Comment (wyli) [inline SwinMM/Pretrain/utils/dataset_in_memory.py:1]: this doesn't look like a python script file though, could you please check the filename?

Comment (wyli) [reply SwinMM/Pretrain/utils/dataset_in_memory.py]: I commented out this line and the line in the file https://github.com/Zi-hao-Wei/research-contributions/blob/5e51052958d8309a9c153d480445f7bb37b1f78c/SwinMM/Pretrain/utils/view_transforms.py please feel free to create new PRs to modify them if I misunderstood... thanks!

**Pull request merged**